### PR TITLE
feat: Verify Customer Form

### PR DESCRIPTION
**Pull request by JoshAltCrd**

### CODE REVIEW

File: src/components/modals/FormalQuestions.vue [hunk 1]
```diff
@@ -0,0 +1,1020 @@
...
+    full() {
+      return this.$route.meta.mode === "full"
+    },
+  },
+
+  created() {
+    $(".tooltip").remove()
+    if (this.customer) this.setCustomer(this.customer)
+    EventBus.$on("customer", customer => this.setCustomer(customer))
+    this.addCustomerOptionsModalsToDom()
```
Comment (darmhoo) [inline src/components/modals/FormalQuestions.vue:906]: Remove this. It has no use. It is causing console error

File: src/components/modals/InformalQuestions.vue [hunk 2]
```diff
@@ -0,0 +1,1015 @@
...
+    full() {
+      return this.$route.meta.mode === "full"
+    },
+  },
+
+  created() {
+    $(".tooltip").remove()
+    if (this.customer) this.setCustomer(this.customer)
+    EventBus.$on("customer", customer => this.setCustomer(customer))
+    this.addCustomerOptionsModalsToDom()
```
Comment (darmhoo) [inline src/components/modals/InformalQuestions.vue:901]: Remove this too

**Review by darmhoo — CHANGES_REQUESTED** [review]

File: src/components/modals/FormalQuestions.vue (hunk 1 above)
Comment (JoshAltCrd) [reply src/components/modals/FormalQuestions.vue:906]: okay

File: src/components/modals/InformalQuestions.vue (hunk 2 above)
Comment (JoshAltCrd) [reply src/components/modals/InformalQuestions.vue:901]: okay

File: src/components/modals/InformalQuestions.vue [hunk 3]
```diff
@@ -0,0 +1,1008 @@
...
+    },
+    
+    data() {
+      return {
+        verifiedOptions: ["Home", "Shop/ Business"],
+        vAgents: null,
+        verificationData: {},
+        locationDurationOptions: ["less than 6 months","6months - 1 year","More than 1 year",],
+        customerWorkingYearsOptions: ["less than 6 months","6months - 1 year","More than 1 year",],
+        customer: "",
```
Comment (darmhoo) [inline src/components/modals/InformalQuestions.vue:881]: remove this

Comment (JoshAltCrd) [reply src/components/modals/InformalQuestions.vue:881]: the customer?

**Review by darmhoo — COMMENTED** [review]
It looks good @teebarg

**Review by darmhoo — APPROVED** [review]

**Review by teebarg — APPROVED** [review]

**Pull request merged**